tealeaf/TeaLeafIOS-Info.plist

```diff
@@ -77,5 +75,11 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>NSPhotoLibraryUsageDescription</key>
+	<string>This app requires access to the photo library.</string>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>This app requires access to your location.</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>This app requires access to your location.</string>
```

Comment:
It is better to move it to the plugin that requires these permissions

Reply:
Will have to find the app which uses this permissions, Do we need to do now? Also I guess we dont need location permission.. right?